CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, sex characteristics, gender identity and expression,
+level of experience, education, socio-economic status, nationality, personal
+appearance, race, religion, or sexual identity and orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
+include:
+
+* Using welcoming and inclusive language
+* Being respectful of differing viewpoints and experiences
+* Gracefully accepting constructive criticism
+* Focusing on what is best for the community
+* Showing empathy towards other community members
+
+Examples of unacceptable behavior by participants include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or
+ advances
+* Trolling, insulting/derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or electronic
+ address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+ professional setting
+
+## Our Responsibilities
+
+Project maintainers are responsible for clarifying the standards of acceptable
+behavior and are expected to take appropriate and fair corrective action in
+response to any instances of unacceptable behavior.
+
+Project maintainers have the right and responsibility to remove, edit, or
+reject comments, commits, code, wiki edits, issues, and other contributions
+that are not aligned to this Code of Conduct, or to ban temporarily or
+permanently any contributor for other behaviors that they deem inappropriate,
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
```

Comment:
The assumption of this part of the document is that there is a clear separation between the personal and the professional. While I really do like that ideal situation, this isn't necessarily true, especially in today's social media era.

There is also the matter of the company's PR. There are many precedents of companies being pressured to punish employees for a perspective they expressed or something that they did outside of the scope of their professional activities, and caving to that pressure. There are notable cases in the tech industry, for example Google vs. Damore and Mozilla's CEO Brendan Eich.

This last line: "Representation of a project may be further defined and clarified by project maintainers" opens the door for ill conceived redefinitions and further trouble.

Reply:
What type of redefinitions and by who? Someone that isn't part of the organization?

Reply:
Redefinitions that include situations other than the ones described here. Could be done by an employee of the organization, with good intentions and having in mind what is best for the team or the company. Regardless, does not change the complexity of the issue and the questionable aspect of the assumption of the scope.